src/cmake-tools.ts

```diff
@@ -605,8 +675,11 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
   /**
    * Implementation of `cmake.debugTarget`
    */
-  async debugTarget(): Promise<vscode.DebugSession|null> {
-    const drv = await this._cmakeDriver;
+  async debugTarget(): Promise<vscode.DebugSession | null> {
+    const drv = await this.getCMakeDriverInstance();
+    if (!drv) {
+      return null;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve(null);
```

Reply:
Style guide question